From 2d84b68bcdcf1325ba15d6d0252894d828a0d765 Mon Sep 17 00:00:00 2001
From: Graham TerMarsch <graham@howlingfrog.com>
Date: Mon, 30 Aug 2021 13:14:00 -0700
Subject: [PATCH] Switch to `nip.io`, from `xip.io` in this test.

`xip.io` stopped working long ago, and `nip.io` provides a similar
service and should be working.  Update test accordingly.

Resolves #4
---
 t/01_simple.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/01_simple.t b/t/01_simple.t
index a121363..65f5132 100644
--- a/t/01_simple.t
+++ b/t/01_simple.t
@@ -78,15 +78,15 @@ subtest "can't do decimal/octal IPs" => sub {
         [ undef, 'DNS lookup resulted in bad host.' ] );
 };
 
-subtest "this domain is okay.  50.112.116.235.xip.io isn't blocked" => sub {
-    is_deeply( [ $dns->resolve('50.112.116.235.xip.io') ],
+subtest "this domain is okay.  50.112.116.235.nip.io isn't blocked" => sub {
+    is_deeply( [ $dns->resolve('50.112.116.235.nip.io') ],
         [ ['50.112.116.235'], undef ] );
 };
 
 subtest 'internal. bad.  blocked by default by module.' => sub {
     is_deeply( [ $dns->resolve('10.2.3.4') ],
         [ undef, 'DNS lookup resulted in bad host.' ] );
-    is_deeply( [ $dns->resolve('50.112.116.235.xip.io') ],
+    is_deeply( [ $dns->resolve('50.112.116.235.nip.io') ],
         [ ['50.112.116.235'], undef ], 'ok' );
 };
 
